v0/util/errors/error-classes.js

```diff
@@ -0,0 +1,80 @@
+// eslint-disable-next-line max-classes-per-file
```

Comment:
Rename this file to `errorTypes.js`.

Reply:
Done. Can you check and let me know if it makes sense ?